v0/destinations/braze/nethandler.js

```diff
@@ -0,0 +1,55 @@
+const { sendRequest } = require("../../../adapters/network");
+const {
+  trimResponse,
+  nodeSysErrorToStatus
+} = require("../../../adapters/utils/networkUtils");
+const { ErrorBuilder } = require("../../util/index");
+
+// eslint-disable-next-line consistent-return
+const responseHandler = (dresponse, metadata) => {
+  if (dresponse.success) {
+    const trimmedResponse = trimResponse(dresponse);
+    const { data } = trimmedResponse;
+
+    if (data.errors && data.errors.length > 0) {
+      throw new ErrorBuilder()
```

Comment:
What's the need for using `new` as the rest of error builder statements chain already return an object of type `Error`?